pkg/proxy/proxy.go

```diff
@@ -436,6 +450,23 @@ func (s *Proxy) addCluster(obj interface{}) {
 		},
 	}
 
+	if cluster.Spec.ExternalKubeAPIEnabled {
+		// get lb annotations from cluster annotations
+		externalLBAnno, ok := cluster.Annotations["tower.kubesphere.io/external-lb"]
```

Comment:
`external-lb` is less descriptive, how about `tower.kubesphere.io/external-lb-service-annoations`

Reply:
changed